OutOfSchool/OutOfSchool.BusinessLogic/Services/CurrentUserService.cs

```diff
@@ -67,12 +65,14 @@ public bool HasClaim(string type, Func<string, bool>? valueComparer = null) =>
         Role.RegionAdmin => IsInRole("regionadmin"),
         Role.AreaAdmin => IsInRole("areaadmin"),
         Role.Moderator => IsInRole("moderator"),
+        Role.Employee => IsInRole("employee"),
         _ => throw new NotImplementedException("Role not handled"),
     };
 
-    public bool IsDeputyOrProviderAdmin() =>
-        IsInRole(Role.Provider) &&
-        (IsInSubRole(Subrole.ProviderDeputy) || IsInSubRole(Subrole.ProviderAdmin));
+    public bool IsEmployeeOrProvider() => IsInRole(Role.Provider) || IsInRole(Role.Employee);
+
+    public bool isProvider() => IsInRole(Role.Provider);
```

Comment:
the following 2 methods were not added to interface. Are they really needed then?